bad.py

```diff
@@ -0,0 +1,4 @@
+5 == 5
+
+concat_list = ["str1" "str2", "str3"]
+6 == 6
```

Comment:
# <img src="https://r2c.dev/semgrep_icon.png" height=20px alt="" /> semgrep
**Severity:** Error :red_circle:
**Rule id:** [python.lang.correctness.useless-eqeq.useless-eqeq](https:// /r/python.lang.correctness.useless-eqeq.useless-eqeq)

This is always True: `6 == 6` or `6 != 6`. If testing for floating point NaN, use `math.isnan(6)`,
or `cmath.isnan(6)` if the number is complex.

<sub>[Report an issue with this rule.](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=python.lang.correctness.useless-eqeq.useless-eqeq)</sub>

Reply:
## <img src="https://r2c.dev/semgrep_icon.png" height=20px alt="" /> semgrep

This is always True: `6 == 6` or `6 != 6`. If testing for floating point NaN, use `math.isnan(6)`,
or `cmath.isnan(6)` if the number is complex.

<sub>[Suppress finding](https://semgrep.dev/docs/running-rules/#managing-findings) | [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=python.lang.correctness.useless-eqeq.useless-eqeq) | [semgrep.dev](https://semgrep.dev/)</sub>

Reply:
## <img src="{LOGO_URL}" height=16px alt="" /> semgrep
This is the message. [Learn more](https://semgrep.dev)
<sub> [Suppress finding](https://semgrep.dev/docs/running-rules/#managing-findings) | [Report a bug](https://semgrep.dev) | [semgrep.dev](https://semgrep.dev)</sub>"